pkg/machine/config.go

```diff
@@ -111,10 +114,46 @@ func (rc RemoteConnectionType) MakeSSHURL(host, path, port, userName string) url
 	return uri
 }
 
+func GetDataHome() (string, error) {
+	if runtime.GOOS == "windows" {
+		home, err := os.UserHomeDir()
+		if err != nil {
+			return "", err
+		}
+		return filepath.Join(home, ".local", ".share"), nil
+	}
+
+	return homedir.GetDataHome()
+}
+
+func GetUserHome() (string, error) {
+	if runtime.GOOS == "windows" {
+		home, err := os.UserHomeDir()
+		if err != nil {
+			return "", err
+		}
+		return home, nil
+	}
+
+	return homedir.Get(), nil
+}
+
+func getConfigHome() (string, error) {
+	if runtime.GOOS == "windows" {
+		home, err := os.UserHomeDir()
+		if err != nil {
+			return "", err
+		}
+		return filepath.Join(home, ".config"), nil
+	}
+
+	return homedir.GetConfigHome()
+}
+
 // GetDataDir returns the filepath where vm images should
 // live for podman-machine
 func GetDataDir(vmType string) (string, error) {
-	data, err := homedir.GetDataHome()
+	data, err := GetDataHome()
```

Comment:
That looks like we'd diverge from `c/storage/pkg/homedir`. Could we update that package instead?